qemu/tests/qemu_guest_agent.py

```diff
@@ -802,29 +802,39 @@ def _action_before_fsfreeze(self, *args):
 
     @error_context.context_aware
     def _action_after_fsfreeze(self, *args):
-        error_context.context("Verfiy FS is frozen in guest.")
-        if not isinstance(args, tuple):
-            return
+        error_context.context("Verfiy FS is frozen in guest.", logging.info)
 
         if not self._open_session_list:
-            raise error.TestError("Could not find any opened session")
+            self.test.error("Could not find any opened session")
         # Use the last opened session to send cmd.
         session = self._open_session_list[-1]
+
         try:
             session.cmd(self.params["gagent_fs_test_cmd"])
         except aexpect.ShellTimeoutError:
-            logging.debug("FS freeze successfully.")
+            logging.info("FS is frozen,can't write in guest.")
```

Comment:
batter add "it is expected" in logging.info, so will not misunderstand when checking log.

Reply:
thanks for your advice, update.
@kimi1978wy  help to check it again,thx.